src/components/Graph.js

```diff
@@ -56,6 +62,29 @@ class Graph extends Component {
 							display: true,
 							padding: 25,
             	text: `Value of ${this.props.crypto} over the past 24 hours.`
+						},
```

Comment:
I changed some of the styling in the graph options object to make it easier to understand/read for the user.  I originally did not optimize the coloring of the graph information, and now its an overall better experience.
The `ticks` on the axes' are now more visible, the `gridLines` are also a lighter color and more visible to the user.
Lastly the `legend label` is consistent with the coloring of the app.